src/App.css

```diff
@@ -106,6 +106,27 @@
   align-self: flex-end;
 }
 
+/* Footer Component */
+footer {
+  margin-top: 3rem;
```

Comment:
the margin-top works well on mobile but on desktop it looks a bit odd that the footer is floating in the middle of the screen when there are only a couple of cards. Is it possible to get the footer to sit at the bottom of the screen when on larger devices? 

<img width="955" alt="image" src="https://user-images.githubusercontent.com/57045550/177009042-73ed9af1-e738-45c9-b418-c49aec388996.png">


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
yes it is pretty much possible and it's resolved now ✔